src/main.py

```diff
@@ -19,17 +18,35 @@
 
 # Set up the Discord client. Connecting to Discord is done at
 # the bottom of this file.
-client = Client(intents=intents)
+if config.testing_server:
```

Comment:
Can we replace `testing_server` with `testing_guild`? As we use the "guild" terminology elsewhere.

Reply:
I did my best to rename in the bizarre Discord tradition of calling them servers for users and guilds for developers, renaming all code references of "server" to guild (including the env variable) but calling them "server" in the user instructions in the README.